search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -98,3 +98,95 @@ function displayTopResultsForCurrentTrend(countryCode) {
   }
   resultElement.innerHTML += '<i>Last updated on ' + date + '</i>';
 }
+
+/**
+ * Sets trends under the top-trends modal tab for the selected country.
+ * @param {string} countryCode The two-letter code of the selected country.
+ */
+function setCountryTrends(countryCode) {
+  const topTrendsTab = document.getElementById('top-trends-tab');
+  topTrendsTab.innerHTML = '';
+  fetch('/country-trends/' + countryCode).then(countryTrends =>
+      countryTrends.json()).then(trends => {
+        if (trends.length === 0) {
+          topTrendsTab.innerText = 
+              'Trends are not available for the selected country.';
+        } else {
+          trends.forEach(trend => {
+            const trendHeader = document.createElement('h5');
+            trendHeader.innerText = trend.topic;
+            topTrendsTab.appendChild(trendHeader);
+          });
+        }
+      });
+}
+
+/** 
+ * Displays the popularity timeline in a country for the current search topic 
+ * in the modal.
+ * @param {string} countryCode The two-letter code of the selected country.
+ */
+function displayPopularityTimeline(countryCode) {
+  const popTimelineElement = document.getElementById('pop-timeline-tab');
```

Comment:
Change the function to be name popularityTimelineElement. I don't think pop is obvious enough.

Reply:
I believe you meant the variable name and element id?
But yes, rename those.